index.rst

```diff
@@ -2,8 +2,8 @@
 LSST DM Developer Guide
 #######################
 
-This is an internal guide for `LSST DM <https://www.lsst.org/about/dm>`_ staff.
-It's also openly available so that others can understand how we're building the LSST's data management subsystem.
+This is an internal guide for `LSST DM <https://www.lsst.org/about/dm>`_ staff during project construction, and Rubin Observatory Data Production during operations. 
```

Comment:
Should it say Data Production staff (since it says DM staff)  ... need our about pages for ops...